### PR TITLE
fix(integrations): correct domain in the install link

### PR DESCRIPTION
**Pull request by oioki**
Fixes this mismatch between an org selected for integration and the current org (in the URL):
image

The install link should contain `https://sentry.sentry.io` in this case.

### CODE REVIEW

**Comment by evanpurkhiser** [comment]
nit: Can you adjust the commit message / PR title to match our commit style? https://develop.sentry.dev/commit-messages/

File: static/app/utils.tsx [hunk 1]
```diff
@@ -339,3 +339,15 @@ export function escapeDoubleQuotes(str: string) {
 export function generateBaseControlSiloUrl() {
   return ConfigStore.get('links').sentryUrl || '';
 }
+
+export function generateIntegrationInstallLink(orgSlug) {
```
Comment (evanpurkhiser) [inline static/app/utils.tsx:343]: Is there somewhere better we can put this instead of in the most generic util?

File: static/app/utils.tsx [hunk 2]
```diff
@@ -339,3 +339,15 @@ export function escapeDoubleQuotes(str: string) {
...
+  const sentryDomain = window.__initialData.links.sentryUrl.split('/')[2];
+  return (
+    window.location.protocol +
+    '//' +
+    orgSlug +
+    '.' +
+    sentryDomain +
+    window.location.pathname
+  );
```
Comment (evanpurkhiser) [inline static/app/utils.tsx:352]: Let's prefer template strings
```suggestion
  return `${window.location.protocol}//${orgSlug}.${sentryDomain}${window.location.pathname}`;
```

File: static/app/utils.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/utils.tsx:343]: @markstory it feels like we must have something like this no?

Comment (evanpurkhiser) [inline static/app/utils.tsx:343]: Actually this function doesn't seem specific to integrations right? Looks like a general URL translator.

Comment (oioki) [reply static/app/utils.tsx]: You're right. This function just switches the domain to the other org, but keeps the URL pathname the same. Renamed to `generateOrgSlugUrl`, sounds good?

**Review by scefali — APPROVED** [review]

**Pull request merged**